src/user_management/authentication/local.ts

```diff
@@ -15,12 +12,6 @@ export  function SetLocalAuthMethod(app: express.Application) {
                 if(result.isError) return done(result.error)
                 if(!result.value) return done("unable to login as provided user")
 
-
-                const key = new NodeRSA(fs.readFileSync(Config.encryption_key_path));
-                const encryptedPassword = key.encryptPrivate(password, "base64");
-
-                if(encryptedPassword === result.value.password) return done(null, result.value)
-
```

Comment:
No idea why this was still in there. I'd rarely worked with username/passwords before so was encrypting rather than using bcrypt. Whoops.